app/src/app/api/v0/inventory/[inventory]/download/route.ts

```diff
@@ -53,9 +53,6 @@ export const GET = apiHandler(async (req, { params, session }) => {
           {
             model: db.models.GasValue,
             as: "gasValues",
-            include: [
-              { model: db.models.EmissionsFactor, as: "emissionsFactor" },
-            ],
```

Comment:
Sorry this was a bit of miscommunication.

We need to change this whole GasValue block to be inside of a (to be added) block for ActivityValue. So we can get the actual gas values associated with the activity here, and also their emissions factors.